lib/facility_maker.rb

```diff
@@ -0,0 +1,79 @@
+class FacilityMaker
+
+    attr_reader :new_facilities, :new_facility_data
+
+    def initialize(dmv_data)
+        @dmv_data = dmv_data
+        @new_facilities = []
+        @new_facility_data = {}
+    end
+
+    def facility_maker_co
+        if @dmv_data.first[:state] == "CO"
+            data_organizer_co
+        end
+    end    
```

Comment:
Really great use of helper methods here. I wonder if you could combine this method and the `facility_maker_ny` methods though and use the if/else statement to determine which state's data is being fed in and then call the appropriate method.

Reply:
and the MO one!